integration/kube_integration_test.go

```diff
@@ -1695,6 +1695,18 @@ func testKubeExecWeb(t *testing.T, suite *KubeSuite) {
 
 	auth := cluster.Process.GetAuthServer()
 
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
+		var found bool
+		for ks, err := range auth.UnifiedResourceCache.IterateKubernetesServers(context.Background(), "", types.SortBy{Field: services.SortByName}) {
+			if !assert.NoError(t, err) {
```

Comment:
i dislike having to pass this sortBy as an argument but i dont see a way around it. i think at least i want to change `UnifiedResourceCache.getSortTree` to use an empty "sortBy.Field" to default to name, that way we dont have to specify anything https://github.com/gravitational/teleport/blob/master/lib/services/unified_resource.go#L194-L205

Reply:
Yes that's perfect. The only two sort trees we have are type and name anyway. You're implicitly "sorting" by type when calling these iterators so if name is all we care about and it's defaulted now, then yes direction is the only thing we care about. Good idea.